### PR TITLE
New Permission Style & Colorize

### PR DESCRIPTION
**Pull request by KillerOfPie**
- Added permissions and getMethods to simplify checking in other files
- Added Colorize method for later use(so we dont need to use ChatColor every time)

### CODE REVIEW

**Comment by SparklingComet** [comment]
I assume this does not include #9 ?

**Comment by KillerOfPie** [comment]
The two are separate

File: src/main/java/org/shanerx/tradeshop/Utils.java [hunk 1]
```diff
@@ -123,4 +144,10 @@ public boolean containsAtLeast(Inventory inv, Material mat, int amt) {
...
 	}
-}
\ No newline at end of file
+	
+	public String colorize(String msg)
+	{
+		msg = ChatColor.translateAlternateColorCodes('&', msg);
+		return msg;
+	}	
+}
```
Comment (SparklingComet) [inline src/main/java/org/shanerx/tradeshop/Utils.java:153]: Can't believe I haven't wrapped this up earlier. We'll have to replace all calls of ChatColor.translateAlternateColorCodes in any class.

Comment (KillerOfPie) [reply src/main/java/org/shanerx/tradeshop/Utils.java:153]: I know, we can do it later tho

Comment (SparklingComet) [reply src/main/java/org/shanerx/tradeshop/Utils.java:153]: Alright, I'll add it to the todo.

**Comment by SparklingComet** [comment]
Merged PR. Deleting New-Permission-Style

**Pull request merged**